src/ert/analysis/_es_update.py

```diff
@@ -388,9 +391,12 @@ def _load_observations_and_responses(
     obs_mask = np.logical_and(ens_mean_mask, ens_std_mask)
 
     if misfit_process:
-        scaling[obs_mask] *= misfit_preprocessor.main(
-            S[obs_mask], scaled_errors[obs_mask]
-        )
+        for input_group in misfit_process:
+            group = _expand_wildcards(obs_keys, input_group)
```

Comment:
Not sure we want to do the expansion here, or if we want to do it earlier, a bit conflicted.

Reply:
Cant agree with myself, in a potential future we might want to do the expansion when we start the base run model, and not when parsing the configuration, given that we will be input a list of observations to update. So I am actually leaning towards keeping it like this for now. But not sure. If we dont we need to do some context validation after we have finished parsing the config (this is parsed before the observations, so we dont know them yet).